examples/sn-react-component-docs/stories/pickers/ExampleApp.tsx

```diff
@@ -11,7 +11,7 @@ import {
 import React from 'react'
 
 const testRepository = new Repository({
-  repositoryUrl: 'https://dmsservice.demo.sensenet.com',
+  repositoryUrl: 'https://dev.demo.sensenet.com',
```

Comment:
This is not going to work in dev.demo please the `currentPath` parameter in useListPicker.